new-site/src/docs/components/tooltip/code.mdx

```diff
@@ -73,3 +70,5 @@ Note! You can find the full list of properties in the <Link size="M" href="/stor
 | `buttonLabel`  | Aria-label text for the tooltip trigger button.              | `string`                                                                                                                                                                                       | "Tooltip"     |
 | `tooltipLabel` | Aria-label text for the tooltip.                             | `string`                                                                                                                                                                                       | "Tooltip"     |
 [Table 1:Tooltip properties]|
+
+export default ({ children }) => <>{children}</>;
```

Comment:
This is export is not needed

Reply:
- [ x] Fixed tabs not rendering on the Tooltip documentation page